control/autoware_vehicle_cmd_gate/src/vehicle_cmd_gate.cpp

```diff
@@ -457,10 +459,21 @@ void VehicleCmdGate::onTimer()
     }
   }
 
+  const auto CheckTopicContinuity = []<class T>(const T & prev_topic, const T & current_topic) {
+    return (rclcpp::Time(current_topic.stamp) - rclcpp::Time(prev_topic.stamp)).seconds() > 0.0
+             ? current_topic
+             : prev_topic;
```

Comment:
It is better to have INFO message to tell "the mode is changed, but the signal is not received yet".

Reply:
II'll accept your suggestion and check the coding conventions again！

Reply:
I added INFO message. However, depending on the controller's specifications, the topic may only be published when a command is entered. In such cases, INFO messages will continue to appear.
https://github.com/autowarefoundation/autoware.universe/pull/8084/commits/bb4e5133cb20308eb8fa61e134f3934d9b884351